assets/src/design-system/icons/landscape.svg

```diff
@@ -0,0 +1 @@
+<svg fill="none" height="93" viewBox="0 0 100 93" width="100" xmlns="http://www.w3.org/2000/svg"><path d="m33 52-32.5 41h99l-24.5-33-17 22z" fill="#fff" opacity=".2"/><circle cx="79" cy="13" fill="#c4c4c4" opacity=".1" r="13"/></svg>
```

Comment:
I don't think this belongs here - it's not a 32x32px icon, but more like a general image resource. Not sure where it should go, but probably not in here. Maybe just inside the specific component until we find a better spot and/or need it elsewhere?

Reply:
Yep, also thought it might out of place. Will add it to the specific component then -- it'll be removed later anyway.